diff --git a/dist/build/next-config-ts/transpile-config.js b/dist/build/next-config-ts/transpile-config.js
index f61819ce61ef09c6b09e64a734f0d3d64c08e53b..228f52db033bc83059eac23788bdc62ee7d5b33b 100644
--- a/dist/build/next-config-ts/transpile-config.js
+++ b/dist/build/next-config-ts/transpile-config.js
@@ -13,6 +13,7 @@ const _promises = require("node:fs/promises");
 const _requirehook = require("./require-hook");
 const _loadjsconfig = require("../load-jsconfig");
 function resolveSWCOptions(cwd, compilerOptions) {
+    compilerOptions = compilerOptions ?? {};
     const resolvedBaseUrl = (0, _nodepath.join)(cwd, compilerOptions.baseUrl ?? '.');
     return {
         jsc: {
